### PR TITLE
added code of conduct, issue/PR templates

### PR DESCRIPTION
**Pull request by Zethson**
Hi,

a code of conduct is OS development best practices. Copied the nf-core/tools one for @ewels consistency reasons.

Also added issue templates and a PR template, which resembles some nf-core style and a little bit of my own old projects. 

Cheers
Signed-off-by: Zethson <lukas.heumos@posteo.net>

### CODE REVIEW

**Comment by TMiguelT** [comment]
I'm happy with code of conduct, and the PR template also looks good. I'm a little wary of being prescriptive with the issue templates though. As a user I find I'm often asked irrelevant questions, or I can't find a category to suit my issue. At very least I think a question template is needed, and maybe add a "where relevant" clause to the system requirements list (e.g. it's not relevant to ask what Node version the user has if they're using the `pip` version or Docker).

**Comment by Zethson** [comment]
@TMiguelT Perfectly agree. Will do.

**Comment by Zethson** [comment]
@TMiguelT Like this?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+1. Run megaqc with command: '...'
+2. Select: '...'
+3. See error
+
+**Expected behavior**
+
+<!-- A clear and concise description of what you expected to happen. -->
+
+<!-- If required to reproduce
+**System [please complete the following information]:**
```
Comment (TMiguelT) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: Maybe combine this previous line into:
> System [please complete the following information, if relevant to the issue]

Comment (Zethson) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Done. I should have read what I even copied there :)

**Comment by Zethson** [comment]
@TMiguelT Can you retrigger the CI, please?

File: .github/ISSUE_TEMPLATE/general_question.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+name: General question
+about: Ask a question about anything MegaQC
+title: 'Question about MegaQC
```
Comment (TMiguelT) [inline .github/ISSUE_TEMPLATE/general_question.md:4]: This quote wasn't closed, but I've since fixed it in master

**Pull request merged**